nemo/collections/nlp/data/language_modeling/text_memmap_dataset.py

```diff
@@ -205,6 +206,28 @@ def _build_data_from_text(self, text):
         return super()._build_data_from_text(text)
 
 
+class JSONLMemMapDataset(TextMemMapDataset):
+    """
+    Memory-mapped iteration over a JSONL file.
+    """
+
+    def __init__(
+        self, dataset_paths, newline_int=10, header_lines=1, workers=None, tokenizer=None, sort_dataset_paths=True,
```

Comment:
the header_lines should be default zero here otherwise it always skips the first data sample.

Reply:
Good point - I won't change defaults here because BioNeMo needs this to be 1 (cc: @michalivne) I think but I will pass header lines as 0 in our datasets.

Reply:
It is already fixed in this PR: https://github.com/NVIDIA/NeMo/pull/5265